presto-iceberg/src/main/java/com/facebook/presto/iceberg/TableStatisticsMaker.java

```diff
@@ -149,6 +150,7 @@ private TableStatistics makeTableStatistics(IcebergTableHandle tableHandle, Opti
         if (intersection.isNone()) {
             return TableStatistics.builder()
                     .setRowCount(Estimate.of(0))
+                    .setConfidenceLevel(LOW)
```

Comment:
Should we set `HIGH` or `FACT` confidence level for a definite `zero` table statistics?